ph_logger.c

```diff
@@ -1068,45 +778,14 @@ static void ph_logger_stop_cloud(struct pantavisor *pv)
 	ph_logger.range_service = -1;
 }
 
-static void ph_logger_start_local(struct pantavisor *pv, char *revision)
-{
-	if (!pv)
-		return;
-
-	if (ph_logger.log_service == -1) {
-		ph_logger.log_service =
-			ph_logger_start_log_service(pv, revision);
-		if (ph_logger.log_service > 0) {
-			pv_log(DEBUG, "started log service with pid %d",
-			       ph_logger.log_service);
-		} else {
-			pv_log(ERROR, "unable to start log service");
-		}
-	}
-}
-
-static void ph_logger_stop_local(struct pantavisor *pv)
-{
-	if (ph_logger.log_service > 0) {
-		kill_child_process(ph_logger.log_service);
-		pv_log(DEBUG, "stopped log service with pid %d",
-		       ph_logger.log_service);
-	}
-
-	ph_logger.log_service = -1;
-}
-
 void ph_logger_toggle(struct pantavisor *pv, char *rev)
 {
 	if (!pv)
 		return;
 
-	if (pv_config_get_log_capture() && pv_config_get_log_loggers())
-		ph_logger_start_local(pv, rev);
-	else
-		ph_logger_stop_local(pv);
-
-	if (pv_config_get_log_push() && pv_get_instance()->remote_mode)
+	if (pv_config_get_log_push() &&
+	    pv_config_get_log_server_output_file_tree() &&
```

Comment:
I guess filetree works fine with log push. Does singletree works with it? maybe it's good not to call ph_logger_start_cloud if singletree is enabled for now? I say this because of TODO in the PR description

Reply:
Well, I mean what happens if singlefile tree is on, alone or along filetree. Does it breaks the log push? If so, we should avoid starting the pusher if singlefile is true.

I ask this because of this line in the description:

```
Fix pantahub log push when in version 2 (if we want to keep this version).
```